modules/select/Select.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useMemo, useState, useEffect } from "react";
+import { withStyles } from "@diana-ui/base";
+import { IChipInputProps } from "@diana-ui/chip";
+import { IDropdownItem, ISingleDropdownProps } from "@diana-ui/dropdown";
+import { useRegistry } from "@diana-ui/hooks";
+
+export interface IProps extends ISingleDropdownProps<IDropdownItem> {
+  onFilter?: (option: IDropdownItem, text: string) => boolean;
+  inputProps?: IChipInputProps;
+}
+
+const defaultFilter = (option: IDropdownItem, text: string) =>
+  option.text.toLowerCase().includes(text.toLowerCase());
+
+function Select(propsT: IProps) {
+  const { inputProps, items, selectedItem, onFilter, ...props } = propsT;
+  const [values, setValues] = useState(selectedItem ? [selectedItem.text] : []);
```

Comment:
Since `Select` only supports one selected item, why not simplify this logic from an array to a IDropdownItem?